packages/cardano-graphql/src/Schema/types/ProtocolParameters.ts

```diff
@@ -0,0 +1,61 @@
+// Review: do we want to keep protocol parameter types per era?
```

Comment:
Yes, each of the following modelled separately: https://github.com/CardanoSolutions/ogmios/blob/220e0be6a71fe7bbfd649fe2d137044b7141d755/clients/TypeScript/packages/schema/src/index.ts#L497-L502